python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -129,6 +129,25 @@ def get_num_targeted_partitions_by_asset_key(self) -> Mapping[AssetKey, int]:
             for asset_key, subset in self.target_subset.partitions_subsets_by_asset_key.items()
         }
 
+    def get_targeted_partitioned_asset_keys_topological_order(self) -> List[AssetKey]:
+        """
+        Returns a topological ordering of partitioned asset keys targeted by the backfill.
+
+        For keys in the same topological level, the order is arbitrary.
```

Comment:
Will it be deterministic? Otherwise, things might jump around on refreshes?

Reply:
Updated this to sort in alphabetical order for keys on the same level